hello.ahk

```diff
@@ -122,833 +76,754 @@ e - cloze
 x - cut
 y - redo
 z - undo
-    )
```

Comment:
mostly in you code I see that it is using spaces instead of tabs for the indentation. It is more organizable to use tabs as indentation.